content/millennium/r4/clinical/medications/immunization.md

```diff
@@ -69,23 +74,33 @@ Search for Immunizations that meet supplied query parameters:
- `_revinclude`    | No                 | [`token`]     | Provenance resource entries to be returned as part of the bundle. Example:_revinclude=Provenance:target
- `target-disease` | No                 | [`token`]     | The target disease the dose is being administered against.                                 Example: `http://hl7.org/fhir/sid/cvx|213`
-
-Notes:
-
-* The `date` parameter may be provided:
-  * once with a prefix `ge` or `le` representing the earliest date or latest date. (e.g. `date=ge2015-01-01`, `date=le2016-01-01`)
-  * twice with the prefixes `ge`, `le` to indicate a specific range. (e.g. `date=ge2015-01-01&date=le2016-01-01`)
-* The `_revinclude` parameter may be provided once with the value `Provenance:target`. Example: `_revinclude=Provenance:target`
-* The `_revinclude` parameter may be provided in combination with the `_id/patient` parameter. Example: `_id=M17255835,M17255827&_revinclude=Provenance:target`
-* When `_revinclude` is provided in a request to a closed endpoint, the OAuth2 token must include the `user/Provenance.read` scope.
-* **Currently the `patient/Provenance.read` scope is not supported and hence `_revinclude` cannot be utilised for patient persona.**
-* The `target-disease` parameter
-  * Searching by Millennium proprietary codes is not supported
+ `_id`            | Conditionally      | [`token`]     | The logical resource ID associated with the resource. This parameter is required if the `patient` parameter is not used. Example: `M12345`
+ `patient`        | Conditionally      | [`reference`] | The patient for the vaccination record. This parameter is required if the `_id` parameter is not used. Example: `12345`
+ `date`           | No                 | [`date`]      | The date range for the immunization administration or administrations. Example: `date=ge2020-01-01T08:00:00.000Z&date=le2020-01-31T17:00:00.000Z`
+ `target-disease` | No                 | [`token`]     | The target disease that the dose is being administered against. Example: `http://hl7.org/fhir/sid/cvx|213`
+ `_revinclude`    | No                 | [`token`]     | The Provenance resource entries that are returned as part of the bundle.
+
+_Implementation Notes_
+
+- When searching with the `_id` parameter:
+  - It can be provided with either a single reference, or a comma-separated list of references. Example: `_id=M1234` or `_id=M1234,M5678`
+  - It must not be provided with any other parameters, except with `_revinclude` as indicated below.
+- When searching with the `patient` parameter:
+  - It can be provided with only a single reference; a comma-separated list is not supported.
+- When searching with the `date` parameter:
+  - For a single `date` occurrence:
+    - It must be provided with either the `le` or `ge` prefix to search for vaccination record administrations earlier or later than the given date.
+    - The `time` component is optional.
```

Comment:
since time is not a unique field/value, should it still be displayed in code format? "date" is not currently in the sentence above.

Reply:
What formatting feels most appropriate to you here, @amcginn?
I would think we want to highlight that component of the date parameter either way.

Reply:
yeah, having the statement is important.
I _think_ remove the code highlighting for "time" since we don't have it for date in the bullet above.